### PR TITLE
Use new GetRelatedPlace() api for similar places and ranking

### PR DESCRIPTION
**Pull request by shifucun**
The new api is based on statsvar dcid and support query multiple statsvars in the same time. This boost the speed of ranking data fetch.

After this is pushed to prod, we can remove the old cache @Spaceenter 

Also get rid of legacy human readable stats var handling as they are removed from cache. @qlj-lijuan

### CODE REVIEW

File: server/main.py [hunk 1]
```diff
@@ -49,6 +48,13 @@
 GCS_BUCKET = app.config['GCS_BUCKET']
 _MAX_SEARCH_RESULTS = 1000
 
+RANKING_STATS = {
```
Comment (beets) [inline server/main.py:51]: Please add a descriptive comment for this const.

File: server/main.py [hunk 2]
```diff
@@ -274,42 +252,26 @@ def api_ranking(dcid):
...
-            'data': get_related_place(
-                dcid, 'CriminalActivities', 'count', 'measuredValue',
-                pvs_string='crimeType^UCR_CombinedCrime',
-                same_place_type=True, within_place=parent, is_per_capita=True)})
-
-    result['label'] = [
-        'Population', 'Median Income', 'Median Age', 'Unemployment Rate',
-        'Crime per capita']
+        crime_statsvar = {
+            'Count_CriminalActivities_CombinedCrime': 'Crime per capita'}
```
Comment (beets) [inline server/main.py:264]: Should this be part of RANKING_STATS as well?

File: server/services/datacommons.py [hunk 3]
```diff
@@ -62,7 +62,7 @@
     'get_place_obs': '/bulk/place-obs',
     'get_chart_data': '/node/chart-data',
     'get_stats': '/bulk/stats',
-    'get_related_places': '/node/related-places',
+    'get_related_places': '/node/related-locations',
```
Comment (beets) [inline server/services/datacommons.py:65]: Is this also temporary, to support the rollout?

File: server/main.py (hunk 2 above)
Comment (shifucun) [reply server/main.py:265]: This one uses perCapita option while the rest does not. The API does not support mixed options, so need to make a separate call. Made an issue to improve on mixer side: https://github.com/datacommonsorg/mixer/issues/262

File: server/services/datacommons.py (hunk 3 above)
Comment (shifucun) [reply server/services/datacommons.py:65]: Yeah, now there are both /node/related-places and /node/related-locations API and cache data. The old one will be removed after rollout.

File: server/main.py (hunk 1 above)
Comment (shifucun) [reply server/main.py:52]: Done

File: server/main.py (hunk 2 above)
Comment (beets) [reply server/main.py:265]: I see, thanks for the clarification. Could you add a note to that here (or in RANKING_STATS above)

File: server/services/datacommons.py (hunk 3 above)
Comment (beets) [reply server/services/datacommons.py:65]: Cool. I'd prefer it if we kept /node/related-places at the end of the switch. :)

File: server/main.py (hunk 2 above)
Comment (shifucun) [reply server/main.py:266]: Done

File: server/services/datacommons.py (hunk 3 above)
Comment (shifucun) [reply server/services/datacommons.py:66]: okay, added a todo

**Pull request merged**